.github/renovate.json

```diff
@@ -7,6 +7,11 @@
     "go": "1.21"
   },
   "packageRules": [
+    {
+      "matchPackageNames": ["PyO3/maturin-action"],
+      "matchPackagePatterns": ["^PyO3/maturin-action$"],
+      "allowedVersions": "1.42.2"
```

Comment:
Not sure if this should be the commit hash instead since it's for the Action (eg. `52b28abb0c6729beb388babfc348bf6ff5aaff31`)?

Reply:
The version is fine, since Renovate checks version tags.